tests/environment-activate.bats

```diff
@@ -79,17 +79,15 @@ env_is_activated() {
 # `pkgdb lock` with no packages installed fetches a nixpkgs. With a package
 # installed, it also has to evaluate the package set.
 @test "warm up pkgdb" {
-  skip "FIXME: needs --ga-registry flag for 'pkgdb manifest lock'";
   run $FLOX_CLI install -d "$PROJECT_DIR" hello;
   assert_success;
   assert_output --partial "✅ 'hello' installed to environment";
-  NIX_CONFIG="extra-experimental-features = flakes" "$PKGDB_BIN" manifest lock "$PROJECT_DIR/.flox/env/manifest.toml"
-  "$BUILD_ENV_BIN" "$NIX_BIN" "$NIX_SYSTEM" "$PROJECT_DIR/.flox/env/manifest.lock" "$PROJECT_DIR/.flox/run/$PROJECT_NAME.$NIX_SYSTEM" "$ENV_FROM_LOCKFILE_PATH";
+  NIX_CONFIG="extra-experimental-features = flakes" "$PKGDB_BIN" manifest lock --ga-registry "$PROJECT_DIR/.flox/env/manifest.toml"
+  # "$BUILD_ENV_BIN" "$NIX_BIN" "$NIX_SYSTEM" "$PROJECT_DIR/.flox/env/manifest.lock" "$PROJECT_DIR/.flox/run/$PROJECT_NAME.$NIX_SYSTEM" "$ENV_FROM_LOCKFILE_PATH";
```

Comment:
Can we just delete this if we're calling the lock command directly?

Reply:
No because the lock command doesn't realize all the store paths. I'll uncomment and make sure it works in my next activate PR but would like to fix activate ASAP